addons/hr_holidays/views/hr_views.xml

```diff
@@ -122,12 +121,12 @@
                 <field name="current_leave_state"/>
                 <field name="leave_date_from"/>
                 <field name="leave_date_to"/>
-                <field name="is_absent"/>
+                <field name="is_absent"/>   
```

Comment:
You can search if there is a plug-in in your IDE that highlights trailing white spaces to avoid putting some by error :) On VSCode/Codium, the plug-in "Trailing Spaces" does just that
```suggestion
                <field name="is_absent"/>
```